.github/workflows/build.yml

```diff
@@ -78,3 +78,25 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+  deploy:
+    name: Deploy to Azure
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        project: [web, cms]
```

Comment:
Just change these names to m0-frontend and m0-backend to use less conditionals below, although you still need some conditionals because the image tags don't match the app names 😅

Reply:
Yeah, just copied it from above